### ISSUE TITLE
PostProcess

### ISSUE THREAD

**Issue by Twxwx** [IDEA-Research/DAB-DETR#50]
Is it possible that multiple labels of a query are topks during post-processing? How to solve this situation?

**Comment by SlongLiu** [comment]
Yes, it may occur sometime. The post-processing is compatible with the situation, where it will select the same box multi times.
